addons/stock_account/migrations/13.0.1.1/post-migration.py

```diff
@@ -0,0 +1,221 @@
+# Copyright 2020 ForgeFlow <http://www.forgeflow.com>
+# Copyright 2020 Andrii Skrypka
+# Copyright 2021 Tecnativa - Carlos Dauden
+# Copyright 2021 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+
+from openupgradelib import openupgrade
+from odoo import _
+from odoo.tools.float_utils import float_compare, float_is_zero, float_round
+from odoo.addons.base.models.ir_model import query_insert
+
+_logger = logging.getLogger(__name__)
+
+# Declare global variant to avoid that it is passed between methods
+precision_price = 0
+
+
+def _prepare_common_svl_vals(move, product):
+    return {
+        "create_uid": move["write_uid"],
```

Comment:
why not move["create_uid"]?

Reply:
The svl record is created when the status of the movement changes to done, so the most common is that the user who validates the movement is the last one to modify the record.